### PR TITLE
OM-571

### PR DESCRIPTION
**Pull request by Citron-mohammad-Rafi**
Please review this PR and merge it.

### CODE REVIEW

File: force-app/main/default/workflows/Opportunity.workflow-meta.xml [hunk 1]
```diff
@@ -721,6 +721,34 @@
         </criteriaItems>
         <triggerType>onAllChanges</triggerType>
     </rules>
+    <alerts>
```
Comment (darcy-old-git) [inline force-app/main/default/workflows/Opportunity.workflow-meta.xml:724]: Alerts should not be in the middle of rules

**Review by darcy-old-git — CHANGES_REQUESTED** [review]
Please close and resubmit with the alerts and rules grouped with each other in the file

File: force-app/main/default/workflows/Opportunity.workflow-meta.xml (hunk 1 above)
Comment (Citron-mohammad-Rafi) [reply force-app/main/default/workflows/Opportunity.workflow-meta.xml]: @darcy-old-git I have resolved the issue according to your comment, Please review it and merge it. Now alert is not in midddle

Comment (Citron-mohammad-Rafi) [reply force-app/main/default/workflows/Opportunity.workflow-meta.xml]: @rgoyalcitronhygiene Please go through the comment and Please review this PR and merge it.

**Review by rgoyalcitronhygiene — APPROVED** [review]

**Comment by rgoyalcitronhygiene** [comment]
As per offline discussion with Rafi, a new pull request will be raised

**Pull request closed without merging**